docs/topics/wildfly-swarm/docs/reference/fractions/transactions.adoc

```diff
@@ -120,17 +129,20 @@ The total number of nested (sub) transactions created.
 swarm.transactions.number-of-resource-rollbacks:: 
 The number of transactions that rolled back due to resource (participant) failure.
 
+swarm.transactions.number-of-system-rollbacks:: 
+The number of transactions that have been rolled back due to internal system errors.
+
 swarm.transactions.number-of-timed-out-transactions:: 
 The number of transactions that have rolled back due to timeout.
 
 swarm.transactions.number-of-transactions:: 
 The total number of transactions (top-level and nested) created
 
 swarm.transactions.object-store-path:: 
-Denotes a relative or absolute filesystem path denoting where the transaction manager object store should store data. By default the value is treated as relative to the path denoted by the "relative-to" attribute.
+Denotes a relative or absolute filesystem path denoting where the transaction manager object store should store data. By default the value is treated as relative to the path denoted by the "relative-to" attribute. This settings is valid when default or journal store is used. It's not used when jdbc journal store is used.
 
 swarm.transactions.object-store-relative-to:: 
-References a global path configuration in the domain model, defaulting to the JBoss Application Server data directory (jboss.server.data.dir). The value of the "path" attribute will treated as relative to this path. Use an empty string to disable the default behavior and force the value of the "path" attribute to be treated as an absolute path.
+References a global path configuration in the domain model, defaulting to the Application Server data directory (jboss.server.data.dir). The value of the "Object store path" attribute will treated as relative to this path. Undefine this attribute to disable the default behavior and force the value of the "Object store path" attribute to be treated as an absolute path.
```

Comment:
Double quoted values should use backticks

Reply:
Note to self: not easily fixable in our sources.